doc/source/getting_started/accessing_models.rst

```diff
@@ -0,0 +1,109 @@
+.. _accessing_models:
+
+================
+Accessing Models
+================
+
+
+.. note:: Suppose you have started the Xinference server endpoint at ``http://0.0.0.0:8001``. 
```

Comment:
Would it be better to have xinference running at `http://127.0.0.1:9997` ?

Reply:
127.0.0.1 is suitable for local deployment. 
What if I use "http://HOSTNAME:PORT" with placeholders instead?